libs/web/design-system/src/lib/styles/theme.tsx

```diff
@@ -1,12 +1,22 @@
 import { ThemeOptions } from '@mui/material/styles';
 
+
+const colorPalette = {
```

Comment:
In this commit, you had all of the properties with their defaults from MUI mapped out, that was correct and should be added back into the colorPalette


https://github.com/aram-devdocs/daruma-board/pull/19/commits/d2ebd73bce036231f75b7d50d53967acf6151c74